activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -947,6 +917,8 @@ def serialize(value)
 
         private
 
+        QUOTED_TRUE, QUOTED_FALSE = '1', '0'
```

Comment:
Is not better to just use the `Quoting::QUOTED_TRUE` in the code that use these constants and make them public constants but not public API?